controllers/keystoneservice_controller.go

```diff
@@ -330,6 +346,7 @@ func reconcileUser(reqLogger logr.Logger, client *gophercloud.ServiceClient, use
 		createOpts := users.CreateOpts{
 			Name:             username,
 			DefaultProjectID: serviceProjectID,
+			Password:         password,
 		}
 		user, err := users.Create(client, createOpts).Extract()
 		if err != nil {
```

Comment:
not important atm, but if pass for two services the same user we probably need to catch it here to not get an error and just ignore it, or check first if the user is already there and run an update instead of create.